src/cascadia/TerminalControl/ScrollBarVisualStateManager.h

```diff
@@ -20,12 +20,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 {
     struct ScrollBarVisualStateManager : ScrollBarVisualStateManagerT<ScrollBarVisualStateManager>
     {
-        ScrollBarVisualStateManager();
```

Comment:
nit: I think now that you removed the ctor, you should be able to remove the factory too?

Reply:
It still has a constructor - it's just automatically generated by the compiler now. I tested it and it seems like it still needs the factory to exist.